### ISSUE TITLE
[terra-form-select] screen reader is unable to read options in Select component

### ISSUE THREAD

**Issue by BrandonAshbee** [cerner/terra-core#1769]
# Bug Report

## Description

Using a screen reader such as JAWS or Narrator for Windows, the reader is unable to read options in the Select component when navigating through the select options using a keyboard. 

## Steps to Reproduce

1. Open screen reader (JAWS, Narrator for Windows, etc)
2. Using the keyboard tab into a Select dropdown component (https://engineering.cerner.com/terra-ui/#/components/terra-form-select/form-select/select)
3. Use the arrow keys to move through the options of the dropdown
4. Expect the highlighted option to be read by the screen reader

## Expected Behavior

A screen reader should be able to read options in the Select component.

## Environment

* Component Name and Version: terra-form-select 4.13.0
* Browser Name and Version: IE 11
* Node/npm Version: [e.g. Node 8/npm 5]
* Webpack Version:
* Operating System and version (desktop or mobile): Windows 10

**Comment by StephenEsser** [comment]
Closing this as a duplicate of: https://github.com/cerner/terra-core/issues/1887
